Modules/mmapmodule.c

```diff
@@ -124,6 +128,12 @@ typedef struct {
     access_mode access;
 } mmap_object;
 
+/*[clinic input]
+module mmap
+class mmap.mmap "mmap_object *" "clinic_state()->mmap_object_type"
```

Comment:
You don't have to designate real type name in here.
See: https://github.com/python/cpython/blob/a768e12f094a9b14a9a1680fb50330e1050716c4/Modules/sha1module.c#L29 
and
https://github.com/python/cpython/blob/a768e12f094a9b14a9a1680fb50330e1050716c4/Modules/_dbmmodule.c#L56

```suggestion
class mmap.mmap "mmap_object *" "mmap_object_type"
```